src/library/trackset/crate/cratetablemodel.cpp

```diff
@@ -37,23 +36,23 @@ void CrateTableModel::selectCrate(CrateId crateId) {
     // (mixxx_deleted = 0) from the view.
     // They are kept in the database, because we treat crate membership as a
     // track property, which persist over a hide / unhide cycle.
-    QString queryString = QString("CREATE TEMPORARY VIEW IF NOT EXISTS %1 AS "
-                                  "SELECT %2 FROM %3 "
-                                  "WHERE %4 IN (%5) "
-                                  "AND %6=0")
-                          .arg(tableName,
-                               columns.join(","),
-                               LIBRARY_TABLE,
-                               LIBRARYTABLE_ID,
-                               CrateStorage::formatSubselectQueryForCrateTrackIds(crateId),
-                               LIBRARYTABLE_MIXXXDELETED);
+    QString queryString = QString(
+            "CREATE TEMPORARY VIEW IF NOT EXISTS %1 AS "
+            "SELECT %2 FROM %3 "
+            "WHERE %4 IN (%5) "
+            "AND %6=0")
+                                  .arg(tableName,
```

Comment:
Was this done by clang-format? I think we have improved it since than.
I would expect that this has no or 8 space indent to the line above.
like that: 
```
    QString queryString =
            QString("CREATE TEMPORARY VIEW IF NOT EXISTS %1 AS "
                    "SELECT %2 FROM %3 "
                    "WHERE %4 IN (%5) "
                    "AND %6=0")
                    .arg(tableName,
                            columns.join(","),
                            LIBRARY_TABLE,
                            LIBRARYTABLE_ID,
                            CrateStorage::formatSubselectQueryForCrateTrackIds(
                                    crateId),
                            LIBRARYTABLE_MIXXXDELETED);
```

Reply:
The are as some more occurrences in this PR. 

Since this is a code styling PR, I suggest to rebase -i and amend all commits using our pre-commit hock one by one.

Reply:
I was questioning this result as well, see https://github.com/mixxxdj/mixxx/pull/2715#discussion_r415294841 ;)

Reply:
Ah, putting the "QStringLiteral" in a new line fixes it, since the continuation indent is relative to the original ;)